tests/functional/disaster-recovery/regional-dr/test_warning_and_alerting.py

```diff
@@ -348,3 +346,227 @@ def test_rdr_inconsistent_data_warning_alert(
                         avoid_stale=True,
                     )
                     logger.info("Action modal closed successfully")
+
+    @polarion_id("OCS-5304")
```

Comment:
hi @dahorak , @petr-balogh , curious if it is ok to use multiple poparion-id's on one test. It can be handy, although, will reporting, ci and scripts in spreadsheets work for every item?

Reply:
discussed here: https://url.corp.redhat.com/ef24633 
looks like having multiple polarion id's is not the solution that we may use in automation

Reply:
Technically this doesn't work, because it saves only the last polarion ID in the XML junit file.
I tried it with following dummy test case:
```
@polarion_id("AAA")
@polarion_id("BBB")
@polarion_id("CCC")
@polarion_id("DDD")
def test_dahorak():
    ...
```
And the junit xml output file looks like this:
```
    <testcase classname="tests.test_dahorak_dummy" name="test_dahorak" time="1.256">
      <properties>
        <property name="polarion-testcase-id" value="DDD"/>
      </properties>
    </testcase>
```

Reply:
Will handle as part of fixing https://github.com/red-hat-storage/ocs-ci/issues/11728